PSQL_README.md

```diff
@@ -0,0 +1,53 @@
+# How to Install PSQL
+## linux
+
+open the terminal and type in
+```bash
+sudo apt-get update 
+sudo apt-get install postgresql postgresql-contrib
+```
+
+## windows
+Download the installer from 
+    https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+and follow the installation process, if asked fill `5432` in port configuration
+
+# How to Configure the Database
+## linux
+Create `systersdb` database, where `systersdb` might be any suitable name.
+
+```bash
+sudo -i -u postgres
+createuser <anyname e.g. alice> --pwprompt
+psql
+CREATE DATABASE systersdb;
+\c systersdb;
+GRANT ALL PRIVILEGES ON DATABASE systersdb to <the name>;
```

Comment:
tbh - its always better to make the instructions use a standard `systersdb` so somebody copy pasting directly dont end up in trouble.